File: app/src/main/java/com/batache/cars/model/adapter/base/FullSizeModelWithHolder.java
package com.batache.cars.model.adapter.base;

import com.airbnb.epoxy.EpoxyHolder;
import com.airbnb.epoxy.EpoxyModelWithHolder;

public abstract class FullSizeModelWithHolder<T extends EpoxyHolder> extends EpoxyModelWithHolder<T> {
  @Override
  public int getSpanSize(int totalSpanCount, int position, int itemCount) {
    return 2;
  }
}

File: app/src/main/java/com/batache/cars/model/adapter/base/FullSizeModelWithHolder.kt
package com.batache.cars.model.adapter.base

import com.airbnb.epoxy.EpoxyHolder
import com.airbnb.epoxy.EpoxyModelWithHolder

abstract class FullSizeModelWithHolder<T : EpoxyHolder?> : EpoxyModelWithHolder<T>() {
  override fun getSpanSize(totalSpanCount: Int, position: Int, itemCount: Int): Int {
    return 2
  }
}